.pre-commit-config.yaml

```diff
@@ -66,7 +66,7 @@ repos:
   ## version = re.search('mypy==([0-9\.]*)', open("constraints.txt").read())[1]
   ## print(f"#========= FROM constraints.txt: v{version} =========")
   ##]]]
-  #========= FROM constraints.txt: v1.10.0 =========
+  #========= FROM constraints.txt: v1.10.1 =========
   ##[[[end]]]
   rev: v1.10.0  # MUST match version ^^^^ in constraints.txt (if the mirror is up-to-date)
```

Comment:
```suggestion
  rev: v1.10.1  # MUST match version ^^^^ in constraints.txt (if the mirror is up-to-date)
```

Reply:
It seems that this version is not available for pre-commit:
```
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'checkout', 'v1.10.1')
return code: 1
stdout: (none)
stderr:
    error: pathspec 'v1.10.1' did not match any file(s) known to git
```